src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -69,6 +74,7 @@ subroutine init_atm_static(mesh, dims, configs)
  integer, pointer :: supersample_fac
 
  real(kind=RKIND):: lat,lon,x,y
+ real(kind=RKIND):: lat,lon,x,y,z
```

Comment:
This line should replace the line above. As is, the code fails to compile with ifort. I'm going to correct this with a rebased branch before merging.